interface/main/tabs/main.php

```diff
@@ -192,15 +192,14 @@ function isEncounterLocked( encounterId ) {
 
                         prev.css('flex', leftPercentage.toString());
                         next.css('flex', rightPercentage.toString());
-
-                        $(document).on("mouseup", function() {
-                          $('body').css('cursor', priorCursor);
-                            $('.draggable').removeClass('draggable').css('z-index', z_idx);
-
-                            // Deactivate Frame Barrier!
-                            $("#frameBarrier").css("visibility", "hidden");
-                        });
                     }
+                    $(document).on("mouseup", function() {
+                        $('body').css('cursor', priorCursor);
+                        $('.draggable').removeClass('draggable').css('z-index', z_idx);
+                        
+                        // Deactivate Frame Barrier!
+                        $("#frameBarrier").css("visibility", "hidden");
+                    });
```

Comment:
I don't see a change

Reply:
Have moved the deactivation barrier part out of the offset ```if``` condition, so that it runs even for the right side of right most tab.

Reply:
Okay I see the move now. It tests well. Have you got any Ideas on how to improve the tabs?

Reply:
The changes are okay for the PR to be merged right?
What improvements are to be made on the tabs? Is it to be done in this PR?